### ISSUE TITLE
Annotation of local variable

### ISSUE THREAD

**Issue by AppletonByte** [silentsignal/android-param-annotate#2]
Is this script only limited to annotating the parameters and not local variables in which we could not display the value of the local variables when debugging using jdb? Thanks.

**Comment by dnet** [comment]
You're right, it only annotates parameters where we have the necessary metadata because of how Java method signatures work (such as 64-bit double and long types using a pair of registers instead of just one).
